<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lsb</class>
 <widget class="QWidget" name="lsb">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>718</width>
    <height>827</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextBrowser" name="Msg">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>60</y>
     <width>161</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="markPic">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>541</width>
     <height>331</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lsbPic">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>360</y>
     <width>711</width>
     <height>461</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="inputMarkPic">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>0</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>put in markPic</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="bitPos">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>40</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>8</number>
   </property>
   <property name="value">
    <number>8</number>
   </property>
  </widget>
  <widget class="QPushButton" name="operateBit">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>40</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>apply bit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
